src/ipolates.f90

```diff
@@ -101,6 +102,85 @@ subroutine ipolates_grid(ip, ipopt, grid_in, grid_out, mi, mo, km,&
 
   end subroutine ipolates_grid
 
+  !> @brief Special case of ipolates_grib1 when interpolating a single field.
```

Comment:
No need for breif tag on subroutines. The first sentence will automatically be used for the brief description.